packages/templates/src/test/getTemplates.js

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+
+/**
+ * Internal dependencies
+ */
+import getTemplates from '../getTemplates';
+
+describe('getTemplate', () => {
+  it('should return png and webp urls for each page', async () => {
+    const templates = await getTemplates('example.com/');
+
+    templates.forEach((template) => {
+      template.postersByPage.forEach((page, i) => {
+        expect(page).toStrictEqual(
+          expect.objectContaining({
+            png: expect.stringContaining(
+              `example.com/images/templates/${template.slug}/posters/${
+                i + 1
+              }.png`
+            ),
+            webp: expect.stringContaining(
+              `example.com/images/templates/${template.slug}/posters/${
+                i + 1
+              }.webp`
+            ),
+          })
```

Comment:
I think `toStrictEqual()` tests to see if the objects serialize to the same value. So you should be able to remove those `expect.Containing()` s.

this should work I think:
```
        expect(page).toStrictEqual({
            png: `example.com/images/templates/${template.slug}/posters/${
                i + 1
              }.png`,
            webp: `example.com/images/templates/${template.slug}/posters/${
                i + 1
              }.webp`,
        })
```

Reply:
Ah just saw your other PR where you add a `type` property to this object. Ignore this comment